app/helpers/application_helper/toolbar/workflow_repositories_center.rb

```diff
@@ -0,0 +1,101 @@
+                   :send_checked => true,
+                   :klass        => ApplicationHelper::Button::ButtonWithoutRbacCheck
+                 ),
+               ])
+  button_group('workflow_repositories',
+               [
+                 select(
+                   :workflow_repositories_configuration,
+                   nil,
+                   t = N_('Configuration'),
+                   t,
+                   :items => [
+                     button(
+                       :embedded_configuration_script_source_refresh,
+                       'fa fa-refresh fa-lg',
+                       N_('Refresh Selected Workflow Repositories'),
+                       N_('Refresh Selected Workflow Repositories'),
+                       :klass        => ApplicationHelper::Button::EmbeddedWorkflow,
+                       :url          => "repository_refresh",
+                       :confirm      => N_("Refresh selected Workflow Repositories?"),
+                       :enabled      => false,
+                       :url_parms    => 'unused_div',
+                       :send_checked => true,
+                       :onwhen       => "1+"
+                     ),
+                     separator,
+                     button(
+                       :embedded_configuration_script_source_add,
+                       'pficon pficon-add-circle-o fa-lg',
+                       t = N_('Add New Repository'),
+                       t,
+                       :klass        => ApplicationHelper::Button::EmbeddedWorkflow,
+                       :url_parms    => "unused_div",
+                       :send_checked => true
+                     ),
+                     button(
+                       :embedded_configuration_script_source_edit,
+                       'pficon pficon-edit fa-lg',
+                       t = N_('Edit this Repository'),
+                       t,
+                       :klass        => ApplicationHelper::Button::EmbeddedWorkflow,
+                       :enabled      => false,
+                       :onwhen       => "1",
+                       :url_parms    => "unused_div",
+                       :send_checked => true
+                     ),
+                     button(
+                       :embedded_configuration_script_source_delete,
+                       'pficon pficon-delete fa-lg',
+                       t = N_('Remove selected Repositories from Inventory'),
+                       t,
+                       :klass        => ApplicationHelper::Button::EmbeddedWorkflow,
+                       :send_checked => true,
+                       :enabled      => false,
+                       :onwhen       => "1+",
+                       :data         => {'function'      => 'sendDataWithRx',
+                                         'function-data' => {:controller     => 'provider_dialogs',
+                                                             :modal_title    => N_('Delete Repositories'),
```

Comment:
@agrare Do we have validations on the backend to ensure you can't delete a repository that has workflows currently in use?

Reply:
No neither does embedded_ansible